### PR TITLE
chore: check jobs’ success instead of workflow

### PR DESCRIPTION
**Pull request by Avishagp**
#### What does this PR do?
Check latest 2 smoke tests' jobs. If the same jobs failed 2 times in a row, try to re-run the latest smoke test and check the job's conclusion. 

#### Any background context you want to provide?
This ensures CLI alerts become less chatty and more focused on urgent problems Hammer should take care of.

### CODE REVIEW

File: test/cli-alert/src/index.ts [hunk 1]
```diff
@@ -10,20 +10,110 @@ if (!process.env.USER_GITHUB_TOKEN || !process.env.SLACK_WEBHOOK_URL) {
...
+  for (let i = 0; i < jobs1.length; i++) {
+    if (
+      jobs1[i].conclusion === 'failure' &&
+      jobs1[i].conclusion === jobs2[i].conclusion
+    ) {
+      console.log(`Found a job that failed 2 times in a row: ${jobs1[i].name}`);
+      failedJobs.push(jobs1[i].name);
+    }
+  }
+}
```
Comment (JackuB) [inline test/cli-alert/src/index.ts:51]: Comparing by index might be inaccurate here. Don't believe GH guarantees order of the checks. Checking and comparing by names might be safer in this case.

```suggestion
for (const jobName of jobs1.map({name} => name)) {
  if ('failure' === jobs1.find(j => j.name === jobName).conclusion === jobs2.find(j => j.name === jobName).conclusion) {
    failedJobs.push(jobName);
  }
}
```

File: test/cli-alert/src/index.ts [hunk 2]
```diff
@@ -10,20 +10,110 @@ if (!process.env.USER_GITHUB_TOKEN || !process.env.SLACK_WEBHOOK_URL) {
...
+
+async function sendSlackAlert() {
+  console.log('Jobs failed again. Sending Slack alert...');
+  const args: IncomingWebhookDefaultArguments = {
+    username: 'Hammer Alerts',
+    text:
+      'A Smoke Tests Job failed more than 2 times in a row. \n <https://github.com/snyk/snyk/actions?query=workflow%3A%22Smoke+Tests%22|Smoke Tests Results>',
```
Comment (JackuB) [inline test/cli-alert/src/index.ts:58]: Idea: might be handy to also include the name of failed tests in the message. E.g. seeing that all `npm` tests failed could be helpful. I guess you'd have to pass `failedJobs.join(', ')` here

**Review by JackuB — APPROVED** [review]
Nothing blocking, approving

**Review by maxjeffos — APPROVED** [review]
Looks good. Maybe a silly question, but how do we know any of this works?

**Pull request merged**